x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/nlp/tokenizers/BertTokenizer.java

```diff
@@ -251,7 +319,7 @@ public TokenizationResult buildTokenizationResult(List<TokenizationResult.Tokeni
             tokenMap[i] = tokenPositionMapSeq1.get(i - 1);
             i++;
         }
-        tokenIds[i] = vocab.get(SEPARATOR_TOKEN);
+        tokenIds[i] = vocab.get(sepToken);
```

Comment:
The huggingface docs suggest that 2 separator tokens should be used for a pair of input sequences:  `pair of sequences: <s> A </s></s> B </s>` 

https://huggingface.co/docs/transformers/model_doc/mpnet#transformers.MPNetTokenizer.build_inputs_with_special_tokens